mms/README.rst

```diff
@@ -0,0 +1,17 @@
+Multi-model-server example
+==========================
+
+https://github.com/awslabs/multi-model-server/
+
+Assuming you are located in the root of the GluonNLP repo, you can run this
+example via:
+
+```
+pip install --user multi-model-server
+curl https://dist-bert.s3.amazonaws.com/demo/finetune/sst.params -o mms/sst.params
+~/.local/bin/model-archiver --model-name bert_sst --model-path mms --handler bert:handle --runtime python --export-path /tmp
```

Comment:
Does it path always work? `~/.local/bin/model-archiver`

Reply:
In a standard setup yes. This readme is currently intended as developer documentation and may need slight adjustment for non-standard setups.